backend/app/tests/utils/test_db.py

```
@@ -165,28 +187,44 @@ def setup_test_db() -> Engine:
     test_db_url = get_test_db_url()
-    
+
     # 检查并确保我们使用的驱动在当前环境可用
+    driver_name = "postgresql+psycopg"
+
+    # 尝试按优先级导入模块
     try:
-        import psycopg2
-        logger.info("Using psycopg2 driver for database connection")
-        driver_name = "postgresql+psycopg2"
+        import psycopg  # 首选 psycopg (v3)
```

Ruff (0.11.9)

`psycopg` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)